src/guide/render-function.md

```diff
@@ -551,6 +551,22 @@ render () {
 
 [`resolveDirective`](/api/global-api.html#resolvedirective) is the same function that templates use internally to resolve directives by name. That is only necessary if you don't already have direct access to the directive's definition object.
 
+### Built-in Components
+
+The [built-in components](/api/built-in-components.html) `<keep-alive>`, `<transition>`, `<transition-group>`, and `<teleport>` are not registered globally be default. This allows bundlers to perform tree-shaking, so that the components are only included in the build if they are used. However, that also means we can't access them using `resolveComponent` or `resolveDynamicComponent`.
```

Comment:
**suggestion**: Improve clarity of listing out specific built-in components rather than a possible misinterpretation of "all built-in component including"

**suggestion**: Typo correction from `be default` to `by default`

```suggestion
[Built-in components](/api/built-in-components.html) such as `<keep-alive>`, `<transition>`, `<transition-group>`, and `<teleport>` are not registered globally by default. This allows bundlers to perform tree-shaking, so that the components are only included in the build if they are used. However, that also means we can't access them using `resolveComponent` or `resolveDynamicComponent`.
```